draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -863,37 +863,37 @@ https://www.example.org
   </tbody>
 </table>
 
-<section title="Header Field Parsing" anchor="field.parsing">
+<section title="Header Field Line Parsing" anchor="field.parsing">
```

Comment:
Field Line Parsing (and I think we can remove "header" from this and the section above)